notebooks/00_quick_start/dkn_synthetic.ipynb

```diff
@@ -22,13 +22,20 @@
     "- DKN uses an attention module to dynamically calculate a user's aggregated historical representaition.\n",
```

Comment:
<p>Here is a performance comparison of several recommendation models on the complete benchmarking MIND dataset [3]. <strong>Please give a very brief introduction of what and how scale of MIND dataset here, so that readers do not need to open the pdf for a first impression.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
finished